cmd/monaco/main.go

```diff
@@ -44,7 +44,21 @@ func Run(args []string) int {
 	return RunImpl(args, util.NewFileReader())
 }
 
+func containsVersionFlag(args []string) bool {
+	for _, arg := range args {
+		if arg == "--version" {
```

Comment:
you're breaking Go flags convention and the help output of the tool with this! 
(and the documentation you've added)

The flag may come in as `-version` as well. 
Also any reason to specifically not provide a `v` shorthand?

Reply:
`v` is already used by verbose. 

Good catch with golangs funny way to handle arguments.